forum/communications/consumers.py

```diff
@@ -0,0 +1,52 @@
+from asgiref.sync import async_to_sync
+from channels.generic.websocket import JsonWebsocketConsumer
+
+from users.models import CustomUser
+
+
+class ChatConsumer(JsonWebsocketConsumer):
+    def connect(self):
+        self.room_name = self.scope["url_route"]["kwargs"]["room_name"]
+        self.room_group_name = f"chat_{self.room_name}"
+
+        print(self.scope["user"])
+
+        print(self.channel_name)
+
+        async_to_sync(self.channel_layer.group_add)(
+            self.room_group_name, self.channel_name
+        )
+
+        self.accept()
+
+    def disconnect(self, code: int):
+        ...
```

Comment:
The `disconnect` method is not implemented. Properly just use `pass` here or handle group removal here.